content/en/docs/concepts/services-networking/endpoint-slices.md

```diff
@@ -4,38 +4,22 @@ reviewers:
 title: EndpointSlices
 content_type: concept
 weight: 45
+description: >-
+  The EndpointSlice API is the mechanism that Kubernetes uses to let your Service
+  scale to handle large numbers of backends.
```

Comment:
> The EndpointSlice API is an evolution of the Endpoints API to let your Service
  scale to handle large numbers of backends.

Reply:
I'd prefer not to mention Endpoints too much - the aim of the commit that this text comes from is to make EndpointSlice _more prominent_.